src/Netscript/WorkerScript.ts

```diff
@@ -146,6 +146,10 @@ export class WorkerScript {
     }
   }
 
+  logKill(reason: string): void {
```

Comment:
This function is tiny and only used once, just inline the impl into `killWorkerScriptByPid`.